### ISSUE TITLE
Signing out from an event should automatically remove email notifications in regards to that event

### ISSUE THREAD

**Issue by marian-gheorghe** [zugspitze/Hiking-Buddies#45]

**Comment by zugspitze** [comment]
@marian-gheorghe Can you describe more what happened?

**Comment by marian-gheorghe** [comment]
I have been subscribed to event https://www.hiking-buddies.com/routes/events/189/.  I have posted a comment and since, started to receive notifications whenever a followup comment was posted. 
I have then signed out from the event and kept getting all the followup comments notifications. Not sure if this is event bound or comments bound, but I think once you have signed out from an event all the notifications "subscriptions" should be "unsubscribed"

**Comment by marian-gheorghe** [comment]
If I remember correctly this is an user enabled option. However, there should be a way of turning off that option (i.e incoming comments notifications).

**Comment by zugspitze** [comment]
Did you unmute the notification email? There must be a link to do that in mail and you can opt out. Let me know if it's missing or not working, I will fix it.

**Comment by marian-gheorghe** [comment]
Correct, I didn't see it as the word "click", followed by a long link and then by the word "mute" could be troublesome to read on a smaller device where the "mute" word is already on the second or third line. 
Here the "mute" action should be emphasized, so I would reformulate:

Click http://www.hiking-buddies.com/comments/mute/.eJx1U1lzG0UQlo9EseKTEAMx... to mute the comments thread 
to 
To mute the comments thread **click here**  (it stands out better, emphasizing the mute option)

**Comment by marian-gheorghe** [comment]
or "Turn off notifications on comments section by clicking here"

**Comment by c-harding** [comment]
While there is the option to unsubscribe, it seems like this should happen automatically if you unregister.
